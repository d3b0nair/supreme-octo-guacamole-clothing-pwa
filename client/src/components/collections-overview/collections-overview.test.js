import React from "react";
import { render, cleanup } from "@testing-library/react";
import { CollectionsOverview } from "./collections-overview.component";

const mockProps = (otherProps) => ({ ...otherProps, collections: [] });
const component = (mockProps) => <CollectionsOverview {...mockProps} />;

afterEach(cleanup);

describe("CollectionsOverview component", () => {
  it("should mount", () => {
    const { container } = render(component(mockProps()));
    expect(container).toMatchSnapshot();
  });
});
