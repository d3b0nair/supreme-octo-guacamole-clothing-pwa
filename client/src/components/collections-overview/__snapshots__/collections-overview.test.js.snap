// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionsOverview component should mount 1`] = `
<div>
  <div
    class="sc-jrQzAO lesZzk"
  />
</div>
`;
